lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -25,6 +26,16 @@ function requestHandler(request, response) {
   const queryString = requestUrl.search && parseQueryString(requestUrl.search.slice(1));
   let absoluteFilePath = path.join(__dirname, filePath);
 
+  if (filePath === '/') {
+    const fixturePaths = glob.sync('**/*.html', {cwd: __dirname});
+    const html = `
+      <h1>Smoke test fixtures</h1>
+      ${fixturePaths.map(p => `<a href=${p}>${p}</a>`).join('<br>')}
```

Comment:
does this work on windows, like are the paths returned by glob different? 

do we care if it doesn't? probably not

Reply:
🤷‍♂